dbfit-java/mysql/src/main/java/dbfit/environment/MySqlEnvironment.java

```diff
@@ -94,6 +94,8 @@ protected String parseCommandText(String commandText) {
             "TINYINT", "SMALLINT", "MEDIUMINT", "INT", "INTEGER" });
     private static List<String> longTypes = Arrays.asList(new String[] {
             "BIGINT", "INTEGER UNSIGNED", "INT UNSIGNED" });
+    private static List<String> bigIntegerTypes = Arrays.asList(new String[] {
```

Comment:
Should the new group be called unsignedBigIntegerTypes? BIGINT is in the group LONG.